tests/components/media_source/test_models.py

```diff
@@ -0,0 +1,27 @@
+"""Test Media Source model methods."""
+from homeassistant.components.media_source import const, models
+
+
```

Comment:
Let's add some basic tests for the parsing of media source content IDs.

Reply:
Isn't that what [these](https://github.com/home-assistant/core/pull/39305/files#diff-4d0b3ce0ca3e18a5394918f0656792bbR14-R17) are?